app/controllers/admin/visit_requests_controller.rb

```diff
@@ -1,18 +1,38 @@
 module Admin
   class VisitRequestsController < ::Admin::BaseController
-    helper_method :visit_requests, :event, :visitors_ids, :confirmed_ids, :applied_ids,
+    helper_method :form, :visit_requests, :event, :visitors_ids, :confirmed_ids, :applied_ids,
                   :pending_ids, :approved_ids
 
-    before_action :only_supervisor!
+    before_action :only_supervisor!, :set_breadcrumbs
 
-    def index
+
+    def create
+      @form ||= InviteByEmailForm.new(visit_request_params)
+
+      if @form.valid?
+        ::VisitRequest::Invite.call(emails: @form.emails, event: event)
+        redirect_to admin_event_visit_requests_path(event)
+      else
+        render :new
+      end
+    end
+
+    private
+
+    def form
+      @form ||= InviteByEmailForm.new
```

Comment:
@floor114 I can't pass `visit_request_params` here because it will raise an error because of `: invite_by_email_form` missing

Reply:
If you mean that `invite_by_email_form` key isn't in params, then you can do next
`params.fetch(:invite_by_email_form, {}).permit(:emails)`

`ActionController::Parameters` [overrides](https://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-fetch) method `fetch`.

Reply:
True. I was lazy to think 🤔 about it. 
Thanks 🙏